Docs: Manage multi-team access

urbiz-grafana opened the pull request:
Turning https://grafana.com/blog/2025/09/08/managing-access-in-grafana-a-single-stack-journey-with-teams-roles-and-real-world-patterns/ into a doc.

urbiz-grafana committed:
Draft

urbiz-grafana committed:
Fixes

urbiz-grafana committed:
Prettier

urbiz-grafana committed:
Fix?

urbiz-grafana committed:
Fix?

urbiz-grafana committed:
Links

urbiz-grafana committed:
Paths

urbiz-grafana committed:
Link

urbiz-grafana committed:
Images

jtvdez commented on docs/sources/setup-grafana/configure-security/manage-single-access.md:
```diff
@@ -0,0 +1,123 @@
+
+- Implement a unified SSO, establishing clear permissions.
+- Reduce setup and maintenance work, avoiding multi-stack complexity.
+- Centralize configuration and plugin management.
```
```suggestion
- Centralize plugin configuration and management.
```

jtvdez commented on docs/sources/setup-grafana/configure-security/manage-single-access.md:
```diff
@@ -0,0 +1,123 @@
+- Team B handles data engineering and needs autonomy with their own dashboards.
+- Team C is the observability team and the admins of the Grafana stack.
+
+Follow these suggested steps to structure, configure and set permissions to access data in your Grafana instance:
```
```suggestion
Follow these suggested steps to structure, configure, and set permissions to access data in your Grafana instance:
```

jtvdez commented on docs/sources/setup-grafana/configure-security/manage-single-access.md:
```diff
@@ -0,0 +1,123 @@
+
+Follow these suggested steps to structure, configure and set permissions to access data in your Grafana instance:
+
+1. [Before you begin](#before-you-begin)
```
Could this be an H3? Basically:
### Before you begin

jtvdez reviewed (changes requested):
I've made a few comments as first pass - Please reach out if there are any questions etc 😄

jtvdez reviewed (changes requested):
I've made a few comments as first pass - Please reach out if there are any questions etc 😄

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
Link

urbiz-grafana committed:
URIs

urbiz-grafana commented on docs/sources/setup-grafana/configure-security/manage-single-access.md:
```diff
@@ -0,0 +1,123 @@
+    - oss
+title: Manage multi-team access in a single Grafana instance
+weight: 1200
+---
```
@jtvdez @irenerl24 - can you guys check I added the refs correcty? TIA!

jtvdez committed:
add an example ref URI

urbiz-grafana committed:
More refs

urbiz-grafana committed:
Prettier

irenerl24 committed:
Merge branch 'main' into urbiz-single-access

jtvdez committed:
adding ref URIs

Updated links to use reference format for drilldown and add-data-source sections.

jtvdez reviewed (changes requested):
Taking another pass with some additional comments

jtvdez reviewed (changes requested):
Taking another pass with some additional comments

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Jacob Valdez <jacob.valdez@grafana.com>

urbiz-grafana committed:
LBAC

urbiz-grafana committed:
Prettier

sarahleejane commented on docs/sources/setup-grafana/configure-security/manage-single-access.md:
```diff
@@ -0,0 +1,158 @@
+
+To design a [folder](ref:create-folder) setup that helps users quickly understand where to go, what they can access, and what they can manage:
+
+- Create an “Everyone” folder for shared items that all teams can manage.
```
```suggestion
- Create an “Everyone” folder for shared items that all teams can manage (ie. grant them admin access to that folder).
```

sarahleejane commented on docs/sources/setup-grafana/configure-security/manage-single-access.md:
```diff
@@ -0,0 +1,158 @@
+
+After you've made sure the model is working, you can codify it.
+
+You can add any new users to your Grafana instance with an Identity Provider through [SCIM](../../configure-security/configure-scim-provisioning/). Use [role sync](../../configure-security/configure-authentication/saml/configure-saml-team-role-mapping/#configure-role-sync-for-saml) to ensure everyone is automatically assigned to the right team based on their membership, including those with the A ...
```
```suggestion
You can add any new users to your Grafana instance with an Identity Provider through [SCIM](../../configure-security/configure-scim-provisioning/). Use [role sync](../../configure-security/configure-authentication/saml/configure-saml-team-role-mapping/#configure-role-sync-for-saml) to automatically assign users the correct basic role (Viewer, Editor, or Admin) based on their mapped attributes in the IdP..
```

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Sarah Constant <sarahleejane@users.noreply.github.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Sarah Constant <sarahleejane@users.noreply.github.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Sarah Constant <sarahleejane@users.noreply.github.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Sarah Constant <sarahleejane@users.noreply.github.com>

urbiz-grafana committed:
Update docs/sources/setup-grafana/configure-security/manage-single-access.md

Co-authored-by: Sarah Constant <sarahleejane@users.noreply.github.com>

urbiz-grafana committed:
Edit
